test.js

```diff
@@ -37,15 +37,27 @@ test('invalid sequential CNPJs', t => {
   t.is(validate(88888888888888), false)
   t.is(validate(99999999999999), false)
   t.is(validate('00000000000000'), false)
+  t.is(validate(null), false)
```

Comment:
I understand the reasoning for this test, but it should be in the block for "invalid unformatted CNPJs" and not on the "invalid sequential CNPJs".

Honestly I should have added this check before, thanks for catching it. I would like this to be on a separate PR, since it's not related to adding the striping of non digit characters as you propose.